build.gradle

```diff
@@ -81,6 +81,8 @@ contracts {
 checkstyle {
 	toolVersion = "8.7"
 	configFile = new File(rootDir, "checkstyle.xml")
+	maxWarnings = 0
+	maxErrors = 0
```

Comment:
Does this need to be propagated into all projects?

Reply:
I think so, i think errors are uneeded but I put them so that it is explicit